scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -8,7 +8,8 @@ set -e
 #===================================================================================================
 # Option parsing
 
-all_comp=(opencv_req opencv_opt python dev myriad installer cl_compiler)
+dafault_comp=(dev python myriad cl_compiler)
+all_comp=(${dafault_comp[@]} opencv_req opencv_opt)
```

Comment:
do we need dependencies for opencv?

Reply:
yes, since we provide script to download OpenCV under extras folder. OpenCV dependencies wont be installed by default. It was agreed do not supply yet another script for OpenCV dependencies. @mshabunin , @alalek